.gitignore

```diff
@@ -53,3 +53,5 @@ nosetests.xml
 
 # vim swap files
 *.swp
+examples/scara_arm/.vscode/.ropeproject/objectdb
+examples/scara_arm/.vscode/.ropeproject/config.py
```

Comment:
This seems too specific. Please do not add these to the .gitignore. Maybe there are some more general statements for vscode that would be more appropriate.

Reply:
Changed. Now it will ignore VS Code local settings.